packages/block-library/src/quote/index.js

```diff
@@ -16,7 +16,7 @@ import settingsV2 from './v2';
 
 const { name } = metadata;
 
-export { metadata, name };
+export { metadata, name, settingsV2 };
```

Comment:
@geriux wanted to check with you this: in https://github.com/WordPress/gutenberg/pull/25892 we're making v2 the default, so this block will only export `settings` (`settingsV2` will no longer exist). Would it be a problem that in https://github.com/WordPress/gutenberg/pull/25892 we remove `settingsV2`?

Not that in Gutenberg 13, the settings exported by the block at https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/quote/index.js#L56 already consider whether v1 or v2 settings should be exported. Wasn't that enough for your purposes?

Reply:
>Would it be a problem that in https://github.com/WordPress/gutenberg/pull/25892 we remove settingsV2?

No problem at all, actually, let me create a PR using that branch in https://github.com/WordPress/gutenberg/pull/25892 as a base to remove the checks we added in this PR.

> Wasn't that enough for your purposes?

Not really, we get the feature flag on the initialization of the mobile editor so we don't have like a global namespace to check its value, so when registering the blocks we use either V1 or V2.

This was temporary of course so we could ship both versions. I had planned to create a follow-up PR to remove what we added here.

Reply:
I created the PR here https://github.com/WordPress/gutenberg/pull/40439 let me know if once approved I can merge it into your branch.

Reply:
Absolutely! Merge at your convenience (or even push directly to the `update/quote-to-nested` branch if that's easier).